src/sentry/utils/performance_issues/performance_detection.py

```diff
@@ -1005,6 +1020,9 @@ def _maybe_store_problem(self):
             metrics.incr("performance.performance_issue.truncated_np1_db")
             return
 
+        if not self._contains_valid_repeating_query(self.n_spans[0]):
```

Comment:
Maybe good to do
```suggestion
        if not self._contains_valid_repeating_query(self.n_spans[0]):
            metrics.incr("performance.performance_issue.unparametrized_first_span")
```

There's a good chance that the difference in volume will be pretty low for whatever reason (e.g., partially parametrized queries, or whatever) so tracking this explicitly might be good

Reply:
Good idea! Added 👍